README.md

```diff
@@ -78,11 +78,19 @@ or if you want to utilize the route's model hook for data-loading, you could do:
 export default class SomeRoute extends Route {
   @context(PersonService) personService;
 
-  async model() {
-    let response = await fetch('https://swapi.co/api/people/1/');
-    let json = await response.json();
+  constructor() {
+    super(...arguments);
+
+    this.router.on('routeDidChange', (transition) => {
```

Comment:
we could handle this for the user in `withContextualServices`

_but_, why `data`?

Reply:
> but, why data?

`data` is the property name you set on existing examples.
https://github.com/NullVoxPopuli/ember-contextual-services/blob/master/README.md?plain=1#L85